src/main/scala/workflow/Node.scala

```diff
@@ -0,0 +1,24 @@
+package workflow
+
+import org.apache.spark.rdd.RDD
+
+sealed trait Node  {
+  def label: String = {
```

Comment:
I didn't want to just call toString because that would get unwieldy w/ case classes parameterized by DenseMatrices/DenseVectors, but we could do some reflection-esque stuff to extract all 'small' parameters.